preactjsx/mod.ts

```diff
@@ -0,0 +1,81 @@
+import { preact, renderToString } from "./deps.ts";
+import loader from "lume/core/loaders/module.ts";
+import { merge } from "lume/core/utils.ts";
+
+import type { Data, Engine, Helper, Site } from "lume/core.ts";
+
+export interface Options {
+  /** The list of extensions this plugin applies to */
+  extensions: string[] | {
+    pages: string[];
+    components: string[];
+  };
+}
+
+// Default options
+export const defaults: Options = {
+  extensions: [".jsx", ".tsx"],
+};
+
+window.preact ||= preact;
```

Comment:
I think this is not neccesary. The jsx library is imported automaticaly thanks to the jsxImportSource configuration in deno.json